bzt/resources/selenium_extras.py

```diff
@@ -297,67 +296,65 @@ def get_loop_range(start, end, step):
     return range(start, end, step)
 
 
-class FrameManager:
-
-    @staticmethod
-    def switch(frame_name=None):
-        driver = _get_driver()
-        try:
-            if not frame_name or frame_name == "relative=top":
-                driver.switch_to_default_content()
-            elif frame_name.startswith("index="):  # Switch using index frame using relative position
-                driver.switch_to.frame(int(frame_name.split("=")[1]))
-            elif frame_name == "relative=parent":  # Switch to parent frame of the current frame
-                driver.switch_to.parent_frame()
-            else:  # Use the selenium alternative
-                driver.switch_to.frame(frame_name)
-        except NoSuchFrameException:
-            raise NoSuchFrameException("Invalid Frame ID: %s" % frame_name)
-
-
-class WindowManager:
-    def __init__(self):
-        self.windows = {}
-
-    def switch(self, window_name=None):
-        driver = _get_driver()
-        try:
-            if not window_name:  # Switch to last window created
-                driver.switch_to.window(driver.window_handles[-1])
+def switch_frame(frame_name=None):
+    driver = _get_driver()
+    try:
+        if not frame_name or frame_name == "relative=top":
+            driver.switch_to_default_content()
```

Comment:
`switch_to_default_content()` - is deprecated in selenium and may stop working in Selenium 4.
use `switch_to.default_content()` instead.